docs/tasks/stateful-sets/upgrade-pet-sets-to-stateful-sets.md

```diff
@@ -0,0 +1,106 @@
+---
+assignees:
+- janetkuo
+
+---
+
+{% capture overview %}
+This page shows how to upgrade from Pet Sets (Kubernetes cluster at version 1.3-1.4) to Stateful Sets (version >= 1.5).
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* If you don't have Pet Sets in your current cluster, or you don't plan to upgrade your cluster/master to >= 1.5 at this moment, you can skip this task now. 
+
+{% endcapture %}
+
+{% capture steps %}
+
+### Differences between alpha Pet Sets and beta Stateful Sets
+
+Pet Set was introduced as an alpha resource in Kubernetes release 1.3, and was renamed to Stateful Set as a beta resource in 1.5. 
+Here are some notable changes:
+
+* **Stateful Set is the new Pet Set**: Pet Set is no longer available in any Kubernetes release >= 1.5. It becomes beta Stateful Set. To know why the name is changed, see this [discussion thread](https://github.com/kubernetes/kubernetes/issues/27430).
+* **Stateful Set guards against split brain**: Stateful Sets guarantee at-most-one semantics to ensure exactly one instance of each pod running anywhere in a cluster, in order to guard against split brain scenarios with distributed applications. *TODO: Link to doc about fencing*
```

Comment:
The phrase "exactly one" contradicts "at-most-one". To avoid having to say "at most one" twice in the sentence, you could say, "Stateful Sets guarantee at most one pod for a given ordinal index can be running anywhere in the cluster".